datasets/fr/amf_regulatory_sanctions/fr_amf_regulatory_sanctions.yaml

```diff
@@ -35,13 +35,22 @@ publisher:
 data:
   url: https://www.amf-france.org/fr/rest/listing_sanction/91,184,183,325,90,461,89,242,86,462,181/all/all?t=1722860865328&_=1722860865172
   format: json
+tags:
+  - issuer.fr                  # Issuer of the data is a French regulatory authority
```

Comment:
is there any case where this is going to differ from publisher.country at the moment? 

Can whatever tooling will use this just use that for now?